rust/src/util.rs

```diff
@@ -92,3 +94,35 @@ pub static RELATIVE_DISTINGUISHED_NAME: LazyPyImport =
 pub static NAME: LazyPyImport = LazyPyImport::new("cryptography.x509", &["Name"]);
 pub static DIRECTORY_NAME: LazyPyImport =
     LazyPyImport::new("cryptography.x509", &["DirectoryName"]);
+
+pub fn generate_random_bytes() -> Vec<u8> {
+    let mut rng = rand::thread_rng();
+    let nonce_random: u64 = rng.gen_range(0..u64::MAX);
+    let nonce_bytes = nonce_random.to_be_bytes();
+
+    // Remove leading 0
+    let first_non_zero = nonce_bytes
+        .iter()
+        .position(|&x| x != 0)
+        .unwrap_or(nonce_bytes.len() - 1);
+    let result = &nonce_bytes[first_non_zero..].to_vec();
```

Comment:
why do we need this?

Reply:
Since `nonce_random` is a `u64`, `to_be_bytes` will generate a 8 bytes long number : 0 => [0, 0, 0, 0, 0, 0, 0, 0].